yoti_python_sdk/activity_details.py

```diff
@@ -1,16 +1,17 @@
 # -*- coding: utf-8 -*-
-import json
-
 import collections
+import json
 
-from yoti_python_sdk import config
+from yoti_python_sdk.anchor import Anchor
+from yoti_python_sdk import config, attribute
 from yoti_python_sdk.protobuf.v1.protobuf import Protobuf
 
 
 class ActivityDetails:
     def __init__(self, receipt, decrypted_profile=None):
         self.decrypted_profile = decrypted_profile
-        self.user_profile = {}
+        self.user_profile = {}  # will be deprecated in v3.0.0
```

Comment:
Do you mean it will be removed from v3.0.0?
If that's the case, the message should be `deprecated`.

Reply:
Yes you're right, it should be `removed`, guessing that's what you meant with your second line?